test-ts/http_receiver_test.ts

```diff
@@ -83,24 +69,20 @@ describe("HTTP Transport Binding Receiver for CloudEvents", () => {
         specversion
       };
 
-      const headers = {
-        [HEADER_CONTENT_TYPE]: MIME_CE_JSON
-      };
-
-      const event = receiver.accept(headers, payload);
+      const event = receiver.accept(structuredHeaders, payload);
       validateEvent(event, specversion);
     });
 
     it("Binary data returns a CloudEvent", () => {
-      const headers = {
-        [HEADER_CONTENT_TYPE]: DEFAULT_CONTENT_TYPE,
-        [DEFAULT_SPEC_VERSION_HEADER]: specversion,
-        [BINARY_HEADERS_03.ID]: id,
-        [BINARY_HEADERS_03.TYPE]: type,
-        [BINARY_HEADERS_03.SOURCE]: source
+      const binaryHeaders = {
+        "content-type": "application/json; charset=utf-8",
+        "ce-specversion": specversion,
+        "ce-id": id,
+        "ce-type": type,
+        "ce-source": source
```

Comment:
As mentioned, I like the other style before as it prevents typos and allows for autocompletion.